server/config/config.go

```diff
@@ -306,3 +306,7 @@ func GetConfig(v *viper.Viper) Config {
 		},
 	}
 }
+
+func (c Config) ValidateBasic() bool {
+	return c.BaseConfig.MinGasPrices != ""
```

Comment:
I think a part that is missing is allowing the application dev to set a default, this is what was discussed. This pr just errors on noting being set.

Reply:
What about just throwing an error with an explicit message on startup "please set min gas price by app.toml or flag or env var"?

Reply:
having a default by the app developer is a more sane approach, IMO. Also better UX. 

Could you do `baseapp.SetMinGasPrices(cast.ToString(appOpts.Get(server.FlagMinGasPrices)), defaultFee),` that way its always set with something?

Reply:
The approach we agreed upon is:

1. Allow the developer to set a default
2. Error when not set

Which means @amaurym is correct -- we need to error when it's empty and have an error message with good context indicating the user should set a sane default. We need to gracefully exit as early as possible.

Reply:
> baseapp.SetMinGasPrices

Sounds good to me. How we wanna allow app developer to set defaultFee value by flag or env variable?

Reply:
> The approach we agreed upon is:
> 
>     1. Allow the developer to set a default
> 
>     2. Error when not set
> 
> 
> Which means @amaurym is correct -- we need to error when it's empty and have an error message with good context indicating the user should set a sane default. We need to gracefully exit as early as possible.

my bad, misunderstood what was said